sockets.js

```diff
@@ -404,10 +405,14 @@ if (cluster.isMaster) {
 	const channels = new Map();
 	/** @type {Map<string, Map<string, string>>} */
 	const subchannels = new Map();
+	/** @type {WriteStream} */
+	const logger = FS('logs/sockets').createAppendStream();
```

Comment:
Is this once per socket? You should probably append a process ID so you don't have multiple processes fighting to append to the same stream.

Reply:
```suggestion
	const logger = FS(`logs/sockets-${process.pid}`).createAppendStream();
```